pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -65,30 +64,26 @@ stdenv.mkDerivation {
 
   doCheck = false; # requires root
 
+  makeFlags = stdenv.lib.optionals (udev != null) [
+    "SYSTEMD_GENERATOR_DIR=$(out)/lib/systemd/system-generators"
+  ];
+
   # To prevent make install from failing.
-  preInstall = "installFlags=\"OWNER= GROUP= confdir=$out/etc\"";
+  installFlags = [
+    "OWNER="
+    "GROUP="
+    "confdir=$(out)/etc"
+  ];
 
   # Install systemd stuff.
-  #installTargets = "install install_systemd_generators install_systemd_units install_tmpfiles_configuration";
-
-  postInstall =
-    ''
-      substituteInPlace $out/lib/udev/rules.d/13-dm-disk.rules \
-        --replace $out/sbin/blkid ${utillinux}/sbin/blkid
```

Comment:
I think this operation has been lost among the other changes?

Reply:
No, lvm doesn't shell out to blkid anymore - see the corresponding commit message in 7d75a7a81ce67595072fdeea7b789e22af14a3f0:

> The call to the blkid binary in lvm2's 13-dm-disk.rules file disappeared (so we don't need to patch in blkid anymore). lvm seems to rely on udev's internal blkid functionality.

Reply:
Ah, I see, thanks.